configure

```diff
@@ -204,13 +204,11 @@ while [ $# -ne 0 ]; do
     --with-caf=*)
       append_cache_entry CAF_ROOT_DIR PATH "$optarg"
       ;;
-<<<<<<< HEAD
-    --with-flatbuffers=*)
-      append_cache_entry FLATBUFFERS_ROOT_DIR PATH "$optarg"
-=======
     --with-bundled-caf)
       append_cache_entry VAST_USE_BUNDLED_CAF BOOL yes
->>>>>>> origin/master
+      ;;
```

Comment:
There is one remaining conlict in this file. Essentially the same as here, but in the `usage` string.

Reply:
Good catch!